src/main/resources/languages/messages_en.yml

```diff
@@ -151,6 +151,7 @@ messages:
     knight: '&a&oYour Talisman gave you 5 Seconds of Regeneration'
     whirlwind: '&a&oYour Talisman reflected the Projectile'
     wizard: '&a&oYour Talisman has given you a better Fortune Level but maybe also lowered some other Enchantment Levels'
+    hastier: '&a&oYour Talisman gave you haste'
```

Comment:
These messages should have a '.' in the end of the sentence.

Reply:
@LinoxGH I followed the other talismans here, none of them had a `.` at the end.
I could update all of them though, if you'd like (if I do this, in this PR or make a new one for that?)

Reply:
> @LinoxGH I followed the other talismans here, none of them had a `.` at the end.
> I could update all of them though, if you'd like (if I do this, in this PR or make a new one for that?)

Oh, well it's up to you then. The whole language file and item lores need a revise anyway.

Reply:
For this PR Ill keep it like the rest, Ill see if I open a new one to add punctuation to the rest